### DISCUSSION TITLE
WOCHE 02 - Software Requirement Specifications

### DISCUSSION THREAD

**Discussion by DossohMael** [mausio/corgi-shop-doc#4] [Blogs]
In der vergangenen Woche wurde kontinuierlich an der App weiterentwickelt. 
Das Back-End für den aktuellen Sprint wurde erfolgreich abgeschlossen und integriert. 
Hierbei kommt nun eine Datenbank zum Einsatz, in der sämtliche Daten gespeichert werden. 
Das Front-End greift ebenfalls auf diese Datenbank zu und präsentiert die Informationen jetzt fehlerfrei.

Zur Darstellung der Vision der App wurde diese Woche außerdem ein Software Requirements Specifications ([SRS](https://github.com/mausio/corgi-shop-doc/blob/main/srs/srs.md)) Document  erstellt

Danke für Ihre Zeit!

**Comment by kattyterra** [comment]
An sich eine interessante Idee, jedoch fehlen in der SRS die Specific Requirements komplett, und das ist schonmal ein Nachteil. Alle relevanten Anforderungen sollten im SRS-Dokument enthalten sein, damit die Entwickler ein vollständiges Verständnis der Anforderungen haben. Jede Anforderung sollte auch eindeutig beschrieben werden, um Verwirrung zu vermeiden.

**Reply by mausio** [reply]
Vielen Dank für dein Feedback. Du hast zu 100% Recht.

**Comment by Puggingtons** [comment]
Euer Scope und Tech-Stack sehen gut durchdacht aus 👍jedoch wären die Reliability Anforderungen auch sehr interessant :)
